docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.md

```diff
@@ -51,7 +51,9 @@ mongodb://<dbuser>:<dbpassword>@dsxxxxx.mlab.com:xxxxx/YOUR-database-name
 
 You'll need to create a database user. Click on the "Users" tab, then click "Add database user" and create a user.
 
-Now you can create the complete URI, which will have the database user and password you just created.
+> Be careful to not use special symbols like '@' or '/' in your password. They will break your URI.
```

Comment:
I would add "unless escaped, which is awkward."

Reply:
Is there a more developer'y way to caveat this than giving two examples? Is there a name for the characters-needing-escaping?

Reply:
We're talking about characters that require escaping in URLs. I suggest we just recommend sticking to alphanumeric characters.

Reply:
I reworded this to `Stick to alphanumeric characters or be careful to escape symbols when creating a user.` good to go?

Reply:
Yeah